src/auth/OnlyContract.huff

```diff
@@ -0,0 +1,14 @@
+/// @title OnlyContract
+/// @author asnared <https://github.com/abigger87>
+
+/* Reverts if the caller is not a contract using tx.origin */
+/* WARNING: This is an anti-pattern and prevents interoperability */
+/* WARNING: Will need to change once account abstraction is implemented */
+#define macro ONLY_CONTRACT() = takes (0) returns (0) {
+    caller              // [msg.sender]
+    origin              // [tx.origin, msg.sender]
+    eq iszero           // [tx.origin != msg.sender]
+    continue jumpi      // []
+    0x00 0x00 revert    // []
```

Comment:
small nit- follow the `0x00 dup1 revert` pattern

Reply:
fixed